wps/main.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+
+from fastapi import FastAPI, HTTPException, Query, status
+from fastapi.responses import RedirectResponse
+from models import Recommendation
+from services import get_playlists_recommendations
+from starlette.status import HTTP_400_BAD_REQUEST
+
+app = FastAPI()
+
+
+@app.get("/", include_in_schema=False)
+def root():
+    return RedirectResponse(url="/docs")
+
+
+@app.get("/recommendation/city/", response_model=List[Recommendation])
+def recommendation_city_name(
+    city: str = Query(..., description='City name'),
+    state: str = Query(..., description='State code', max_length=2),
+    country: str = Query(
+        ..., description='Country code - ISO 3166', max_length=2
+    ),
+):
+    """
+    Provide the city name, state code and country code (according to
+    ISO 3166) to get playlist recommendations based on that place's
+    current temperature.
+    """
+    location = {'city': city, 'state': state, 'country': country}
+    try:
+        recommendations = get_playlists_recommendations(location)
+        return recommendations
+    except Exception as error:
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST, detail=str(error)
```

Comment:
Always returning 400 is not a good practice because there are many types of errors your API could return and this may confuse the user. For example, if one of the external APIs your service needs, such as the weather API, is out, you should return a 503 instead. 

Other than this, good job with the separation of concern here.